server/utils/caching/memory.js

```diff
@@ -0,0 +1,21 @@
+class InMemoryCachingStrategy {
+  constructor() {
+    this.state = {};
+  }
+
+  async set(key, value, expiresInSeconds) {
```

Comment:
Do we want any defaults or checks for empty values?

Reply:
Added additional checks for parameter values